flocker/cli/script.py

```diff
@@ -250,9 +250,49 @@ def _changestate_on_nodes(self, deployment, deployment_config,
         return DeferredList(results)
 
 
+@flocker_standard_options
+class CLIOptions(Options):
+    """
+    Command line options for ``flocker`` CLI.
+
+    """
+    longdesc = """flocker does nothing (yet).
+
+    """
+
+    synopsis = ("Usage: flocker [OPTIONS] "
+                "\n"
+                "If you have any issues or feedback, you can talk to us: "
+                "http://docs.clusterhq.com/en/latest/gettinginvolved/"
+                "contributing.html#talk-to-us")
+
+
+@implementer(ICommandLineScript)
+class CLIScript(object):
+    """
+    A command-line script to interact with a cluster via the API.
```

Comment:
:question: It seems to be that the key differentiator of this script to the others (apart from `flocker-deploy` which this will replace) is that it is user facing, while the others are run either automatically or by the developers of `Flocker`. This might be a useful thing to document.